app/src/main/java/com/iyanuoluwa/imovie/data/ApiCredits.kt

```diff
@@ -0,0 +1,13 @@
+package com.iyanuoluwa.imovie.data
+
+import com.iyanuoluwa.imovie.api2.CreditsJSon
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface ApiCredits {
```

Comment:
You don't have to implement this now, but all these API methods can (and should) be in just one interface. You don't have to create a new interface whenever you want to call another API endpoint.